std/os/linux.zig

```diff
@@ -259,6 +259,10 @@ pub fn pwrite(fd: i32, buf: [*]const u8, count: usize, offset: usize) usize {
     return syscall4(SYS_pwrite, @bitCast(usize, isize(fd)), @ptrToInt(buf), count, offset);
 }
 
+pub fn copy_file_range(fd_in: i32, off_in: ?*isize, fd_out: i32, off_out: ?*isize, len: usize, flags: u32) usize {
```

Comment:
The use of `isize` for the offsets is wrong, the kernel expects a `loff_t` (a `unsigned long`).

Reply:
note that `unsigned long` in linux-land is equivalent to `usize`

Reply:
> note that unsigned long in linux-land is equivalent to usize

Have you ever heard of the so-called 32bit systems?

Reply:
you mean like i386? I'm pretty sure sizeof(long) is 4 in that case, right? or is this logic incorrect:
https://github.com/ziglang/zig/blob/5954d5235fd0d029034d68f82bb7831ff47506f8/src/target.cpp#L873

Reply:
Oh well, I meant `off_in` and `off_out` and not `isize` heh. [The kernel definition](https://elixir.bootlin.com/linux/v5.1.5/source/fs/read_write.c#L1634) indeed suggests that the size is a `size_t` but the offsets are `loff_t` ([defined here](https://elixir.bootlin.com/linux/v5.1.5/source/include/uapi/asm-generic/posix_types.h#L88)) and are `unsigned long long`. Again, I've misquoted the source code.

Sorry about that, I shouldn't write anything before my morning coffee.

Reply:
Yes, this is a off_t, so it is always 64-bit (except with legacy syscalls).

Reply:
If the kernel definition is `unsigned long long` then `off_in` and `off_out` should be `*u64`.